uphold/uphold-sdk-javascript
# Fixing dependency vulnerabilities

#### Description

These were found via `yarn audit`. Resolutions were added for packages with large amounts of varying sub-versions.

**Before:**
![Screen Shot 2019-08-30 at 1 42 53 PM](https://user-images.githubusercontent.com/8732757/64051643-a0e8f080-cb30-11e9-97ae-cd23e4fcbf31.png)

**After:**
![Screen Shot 2019-08-30 at 2 09 47 PM](https://user-images.githubusercontent.com/8732757/64051647-a3e3e100-cb30-11e9-9614-1a3221867e36.png)



#### Test Plan
1. Pull this branch, run `yarn && yarn audit`
2. Ensure that no vulnerabilities are found after the audit script
3. Confirm that `yarn lint && yarn test` passes as expected
